src/app/modules/shared/time-range-form/time-range-form.component.ts

```diff
@@ -0,0 +1,50 @@
+import { ToastrService } from 'ngx-toastr';
+import { formatDate } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import * as entryActions from '../../time-clock/store/entry.actions';
+
+import {Store} from '@ngrx/store';
+import {EntryState} from '../../time-clock/store/entry.reducer';
+import * as moment from 'moment';
+
+@Component({
+  selector: 'app-time-range-form',
+  templateUrl: './time-range-form.component.html',
+})
+export class TimeRangeFormComponent implements OnInit {
+  public reportForm: FormGroup;
+  private startDate = new FormControl('');
+  private endDate = new FormControl('');
+
+  constructor(private store: Store<EntryState>, private toastrService: ToastrService) {
+    this.reportForm = new FormGroup({
+      startDate: this.startDate,
+      endDate: this.endDate
+    });
+  }
+  ngOnInit(): void {
+    this.setInitialDataOnScreen();
+  }
+
+  setInitialDataOnScreen() {
+    this.reportForm.setValue({
+      startDate: formatDate(moment().startOf('week').toString(), 'yyyy-MM-dd', 'en'),
```

Comment:
you can use a constant and re-use across all the components using dates
`CONST DATE_FORMAT = 'yyyy-MM-dd';`

then:
```
      startDate: formatDate(moment().startOf('week').toString(), DATE_FORMAT, 'en'),
```

I highly encourage you to look at the reports page and apply this change there as well.

Reply:
@rene-enriquez-ct I can use this CONST DATE_FORMAT = 'yyyy-MM-dd'; in the file environment.ts for reuse in many files.

Reply:
We apply this change in others files with exactly  format `yyyy-MM-dd` and we create a new constant  DATE_FORMAT_YEAR for this format` YYYY-MM-DD`